shared/templates/create_audit_rules_unsuccessful_file_modification_detailed.py

```diff
@@ -37,6 +37,53 @@ def generate(self, target, args):
                 },
                 "./oval/audit_rules_unsuccessful_file_modification_{0}_rule_order.xml", syscall
             )
+
+        elif target == "bash":
+            self.file_from_template(
+                "./template_BASH_audit_rules_unsuccessful_file_modification_o_creat",
```

Comment:
Please remove `#remediation = none` from the test scenarios for these rules.

Reply:
@jan-cerny done